backend/docker-compose.test-backend.yml

```diff
@@ -68,6 +68,7 @@ services:
             PULL_REQUEST_NUMBER: ${PULL_REQUEST_NUMBER}
             SEMAPHORE_BUILD_NUMBER: ${SEMAPHORE_BUILD_NUMBER}
             SEMAPHORE: ${SEMAPHORE}
+            HIJACK_LOG_FILTER: ${HIJACK_LOG_FILTER}
```

Comment:
we should add the `docker-compose.test-backend.yml` file on the verify hook as well

Reply:
@slowr yes could you do this directly on master please?